.travis.yml

```diff
@@ -37,3 +58,42 @@ jobs:
           script: unbuffer bash ci/travis/build/rabbit-mq deploy
           on:
             all_branches: true
+
+    - stage: test
+      name: devpi test
+      language: python
+      sudo: required
+      python:
+        - "3.7"
+      cache: pip
+      before_install:
+        - sudo bash ci/travis/tests/devpi before_install || travis_terminate 0
+      install:
+        - unbuffer bash ci/travis/tests/devpi install
+      before_script:
+        - unbuffer bash ci/travis/tests/devpi before_script
+      script:
+        - unbuffer bash ci/travis/tests/devpi script
+      after_success:
+        - unbuffer bash ci/travis/tests/devpi after_success
+
+    - stage: release
+      if: NOT type = pull_request and branch = master
+      name: devpi release
+      language: python
+      sudo: required
+      python:
```

Comment:
Why would you need python? Is not only docker required?

Reply:
I think one can use generic for language. but if you don't set any then you'll get ruby which also looks weird ;)

Reply:
I changed for miminal, who doesn't have any language in particular (https://docs.travis-ci.com/user/languages/minimal-and-generic/)